.github/workflows/watch-conda-forge.yml

```diff
@@ -0,0 +1,104 @@
+# Check for new versions of dask on Conda Forge
+name: WatchCondaForge
+
+on:
+  schedule:
+    - cron: "0 * * * *"
+
+jobs:
+  check-version:
+    runs-on: ubuntu-latest
+    if: github.repository == 'conda-forge/pangeo-dask-feedstock'
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: calver
+        id: calver
+        run: echo "::set-output name=version::$(date +'%Y.%m.%d')"
+
+      - name: Get latest dask version
+        id: latest_version_dask
+        uses: jhamman/gha-anaconda-package-version@feature/calver
```

Comment:
these all use semver right, so we can stick with jacobtomlinson/gha-anaconda-package-version@0.1.1 ?